src/assets/styles/_icons.scss

```diff
@@ -0,0 +1,96 @@
+  font-weight: normal;
+  font-style: normal;
+}
+
+// https://sass-lang.com/documentation/values/maps
+$icons: (
+    play: 'f101',
+    play-circle: 'f102',
+    pause: 'f103',
+    volume-mute: 'f104',
+    volume-low: 'f105',
+    volume-mid: 'f106',
+    volume-high: 'f107',
+    fullscreen-enter: 'f108',
+    fullscreen-exit: 'f109',
+    spinner: 'f10a',
+    subtitles: 'f10b',
+    captions: 'f10c',
+    hd: 'f10d',
+    chapters: 'f10e',
+    downloading: 'f10f',
+    file-download: 'f110',
+    file-download-done: 'f111',
+    file-download-off: 'f112',
+    share: 'f113',
+    cog: 'f114',
+    square: 'f115',
+    circle: 'f116',
+    circle-outline: 'f117',
+    circle-inner-circle: 'f118',
+    cancel: 'f119',
+    repeat: 'f11a',
+    replay: 'f11b',
+    replay-5: 'f11c',
+    replay-10: 'f11d',
+    replay-30: 'f11e',
+    forward-5: 'f11f',
+    forward-10: 'f120',
+    forward-30: 'f121',
+    audio: 'f122',
+    next-item: 'f123',
+    previous-item: 'f124',
+    shuffle: 'f125',
+    cast: 'f126',
+    picture-in-picture-enter: 'f127',
+    picture-in-picture-exit: 'f128',
+    close: 'f129',
+    facebook: 'f12a',
+    linkedin: 'f12b',
+    twitter: 'f12c',
+    tumblr: 'f12d',
+    pinterest: 'f12e',
+    audio-description: 'f12f',
+    cart: 'f130',
+);
+
+// NOTE: This is as complex as we want to get with SCSS functionality.
+//
+// Now that we have a map of icons above, we can iterate over that map and create an icon class
+// for each icon in that list. The iterator below produces CSS classes like this:
+//
+// .vjs-icon-play {
+//   font-family: VideoJS;
+//   font-weight: normal;
+//   font-style: normal;
+// }
+// .vjs-icon-play:before { content: "\f101"; }
+//
+// We can then use @extend in the codebase when we need to add an icon to a class. @extend builds up
+// the selectors for you so you can avoid duplication. This is generally a bad idea, but since each
+// icon should only be extended one or two other places, we'll roll with it.
+@each $name, $content in $icons {
+  .vjs-icon-#{$name} {
+    font-family: $icon-font-family;
+    font-weight: normal;
+    font-style: normal;
+
+    &:before {
+      content: char($content);
+    }
+  }
+}
```

Comment:
was this file added automatically?

Reply:
It was generated automatically, but added by me.
I added a short readme explaining how to update and generate this icon-font